### DISCUSSION TITLE
Where are you Leo Khoa? Is everything ok?

### DISCUSSION THREAD

**Discussion by Paran360** [leokhoa/laragon#890] [General]
The sponsor @alxndr-w  has been trying to contact the author/maintainer with no success. Has something happened? 
@leokhoa  **Are you okay brother?**

**Comment by AWqxKAWERbXo** [comment]
Maybe someone can try to reach him via https://[x.com/leovn](https://x.com/leovn)

**Comment by Sophist-UK** [comment]
[The last post @leokhoa made in these discussions that I have found was September 3rd](https://github.com/leokhoa/laragon/discussions/728#discussioncomment-10536921) which was only about 6 weeks ago.

Hopefully he is still OK.
